proposals/exception-handling/Exceptions-formal-overview.md

```diff
@@ -222,6 +223,12 @@ S;C, labels (catch [t*]) ⊢ instr* : []→[t*]
 S;C, labels [t*] ⊢ caught{a val^n} instr* end : []→[t*]
 ```
 
-## TODO Uncaught Exceptions
+## Uncaught Exceptions
+
+A new [result](https://webassembly.github.io/spec/core/exec/runtime.html#syntax-result) value is added to describe uncaught exceptions.
+
+```
+result ::= val* | trap
+         | T[val* (throw tagaddr)]
```

Comment:
- What @rossberg suggested was `val* T[throw e] instr*`... Is this different from `T[val* (throw tagaddr)]`?
- How are the cases, which a throw context contains not only a `throw` and `val`s but other things... such as `label`, `caught`, or `frame` as listed in the throw context section, covered?

Reply:
> What @rossberg suggested was `val* T[throw e] instr*`... Is this different from `T[val* (throw tagaddr)]`?

They were different when @rossberg suggested the former formulation, but in this PR I also changed throw contexts from `T ::= val* '[_]' instr* | ...` to `T ::= '[_]' | val* T instr* | ...`, so now these two things `val* T[throw e] instr*` and `T[val* (throw tagaddr)]` are really the same. 

>  How are the cases, which a throw context contains not only a throw and vals but other things... such as label, caught, or frame as listed in the throw context section, covered?

So now throw contexts surrounded by values and instructions are also throw contexts. Meaning that `val* T instr*` is a throw context too.